index.md

```diff
@@ -17,24 +17,175 @@ redirect_from:
+The [DNS flag day 2019](/2019/) was very successful event and the Internet
+community worked together and fixed problems which were causing delays and
+other problems for Internet users worldwide. We would like to thank to all
 operators who cooperated and helped to make Internet a better place.
 
 Summary of the past and future DNS flag days can be found e.g. in
 [https://youtu.be/mH_elg9EUWw?t=649](https://youtu.be/mH_elg9EUWw?t=649).
 
+Contents
+========
+- [What's next?](#whats-next)
+- [DNS Flag Day 2020](#dns-flag-day-2020)
+  - [Note: Work in progress](#note-work-in-progress)
+  - [Action: Authoritative DNS Operators](#action-authoritative-dns-operators)
+  - [Action: DNS Resolver Operators](#action-dns-resolver-operators)
+  - [Action: DNS software vendors](#action-dns-software-vendors)
+  - [How to test?](#how-to-test)
+- [Previous flag days](#previous-flag-days)
+- [Who's behind DNS flag day?](#whos-behind-dns-flag-day)
+- [Get in touch](#get-in-touch)
+- [FAQ](#faq)
+
 What's next?
 ============
 
-Next round of DNS flag day is being planned right now, with focus on
+The next DNS flag day is being planned right now. It will focus on the
 operational and security problems in DNS caused by Internet Protocol
+packet fragmentation.
+
+Please subscribe to [the dns-announce mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-announce)
+or follow [@dnsflagday on Twitter](https://www.twitter.com/dnsflagday)
+to receive a notification when more information becomes available.
+
+DNS Flag Day 2020
+=================
+
+The DNS community has been discussing persistent interoperability and
+performance issues with the DNS system on industry mailing lists and at
+conferences such as [DNS-OARC 30](https://www.dns-oarc.net/oarc30) panel
+discussion ([video](https://youtu.be/mH_elg9EUWw?t=680),
+[slides](https://indico.dns-oarc.net/event/31/contributions/678/attachments/673/1102/dns_flag_day_panel.pdf)).
+
+The proposed plan for the DNS flag day 2020 was announced at
+[RIPE78](https://ripe78.ripe.net) by Petr Špaček, CZ.NIC and Ondřej Surý,
+ISC ([video](https://ripe78.ripe.net/archives/video/28),
+[slides](https://ripe78.ripe.net/presentations/53-plenary.pdf)). This time,
+we will focus on the problems with IP fragmentation of DNS packets.
+
+IP fragmentation is a problem on the Internet today, especially when it
+comes to large DNS messages. And even if fragmentation works it might not be
+secure enough for DNS.
+- https://tools.ietf.org/html/draft-bonica-intarea-frag-fragile-03
+- http://www.potaroo.net/ispcol/2017-08/xtn-hdrs.html
+- Research by Kazunori Fujiwara https://indico.dns-oarc.net/event/31/contributions/692/
+
+Most of these issues can be fixed by honoring an EDNS buffer size that will
+not cause fragmentation and by allowing DNS to switch from UDP to TCP when
+larger buffer sizes are not enough.
+
+Note: Work in progress
+----------------------
+
+This web site and some aspects of DNS flag day 2020 are work in progress.
```

Comment:
The exact date for the 2020 DNS Flag Day is not yet determined, and the exact recommended EDNS buffer sizes have not been agreed upon (the current consensus is that a buffer size of 1220 - 1232 will work best to limit the risk of fragmentation in IPv6. 
Nevertheless, the technical requirements are already clear enough that operators and developers can start preparing by testing and fixing their systems.

Reply:
You need to review this as a `suggestion`, can't really see what you changed.

Reply:
I was just trying to improve the English for the first 4 sentences. I am not sure what i did wrong here.